src/Apps/Artist/Routes/Overview/Components/RecommendedArtist.tsx

```diff
@@ -0,0 +1,179 @@
+import { EntityHeader, Sans, Spacer } from "@artsy/palette"
+import { RecommendedArtist_artist } from "__generated__/RecommendedArtist_artist.graphql"
+import { track } from "Artsy/Analytics"
+import * as Schema from "Artsy/Analytics/Schema"
+import { SystemContext } from "Artsy/SystemContext"
+import FillwidthItem from "Components/Artwork/FillwidthItem"
+import { FollowArtistButtonFragmentContainer as FollowArtistButton } from "Components/FollowButton/FollowArtistButton"
+import { Carousel } from "Components/v2"
+import { stringify } from "qs"
+import React, { FC, useContext } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { data as sd } from "sharify"
+import { get } from "Utils/get"
+
+interface RecommendedArtistProps {
+  artist: RecommendedArtist_artist
+}
+const HEIGHT = 150
+
+const handleOpenAuth = (mediator, artist) => {
+  if (sd.IS_MOBILE) {
+    openMobileAuth(artist)
+  } else if (mediator) {
+    openDesktopAuth(mediator, artist)
+  } else {
+    window.location.href = "/login"
+  }
+}
+
+const openMobileAuth = artist => {
+  const params = stringify({
+    action: "follow",
+    contextModule: Schema.ContextModule.RecommendedArtists,
+    intent: "follow artist",
+    kind: "artist",
+    objectId: artist.id,
+    signUpIntent: "follow artist",
+    trigger: "click",
+    entityName: artist.name,
+  })
+  const href = `/sign_up?redirect-to=${window.location}&${params}`
+
+  window.location.href = href
+}
+
+const openDesktopAuth = (mediator, artist) => {
+  mediator.trigger("open:auth", {
+    mode: "signup",
+    copy: `Sign up to follow ${artist.name}`,
+    signupIntent: "follow artist",
+    afterSignUpAction: {
+      kind: "artist",
+      action: "follow",
+      objectId: artist.id,
+    },
+  })
+}
+
+@track({
+  context_module: Schema.ContextModule.RecommendedArtists,
+})
+class RecommendedArtistWithTracking extends React.Component<
+  RecommendedArtistProps
+> {
+  @track({
+    type: Schema.Type.Thumbnail,
+    action_type: Schema.ActionType.Click,
+  })
+  trackArtworkClicked() {
+    // noop
+  }
```

Comment:
Methods shouldn’t be called “track” something. Even when we define no-ops right now, the fact that we attach tracking hooks to a lifecycle callback should remain an implementation detail. In this case the method should probably just be `handleArtworkClick`.